### PR TITLE
[FE](fix): improve drag and sort [GMW-756]

### PR DESCRIPTION
**Pull request by anamontiaga**
### Feature relevant tickets
[GMW-756](https://vizzuality.atlassian.net/jira/software/c/projects/GMW/boards/110/backlog?selectedIssue=GMW-756)

[GMW-756]: https://vizzuality.atlassian.net/browse/GMW-756?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/containers/map/index.tsx [hunk 1]
```diff
@@ -367,7 +367,7 @@ const MapContainer = ({ mapId }: { mapId: string }) => {
                 onSetCustomPolygon={handleCustomPolygon}
               />
             )}
-            <Controls className="absolute bottom-9 right-6 items-center print:hidden">
+            <Controls className="absolute bottom-9 right-6 hidden items-center print:hidden md:block">
```
Comment (mluena) [inline src/containers/map/index.tsx:370]: hidden??

File: src/containers/navigation/index.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import News from 'containers/navigation/news';
 const HELPER_ID = 'menu-categories';
 
 const AppTools = () => (
-  <div className="fixed top-2 left-4 z-20 h-11 w-[526px] rounded-4xl bg-brand-800 px-5 shadow-md print:hidden">
+  <div className="fixed top-2 left-4 z-20 hidden h-11 w-[526px] rounded-4xl bg-brand-800 px-5 shadow-md print:hidden md:block">
```
Comment (mluena) [inline src/containers/navigation/index.tsx:10]: hidden?

File: src/containers/map/index.tsx (hunk 1 above)
Comment (anamontiaga) [reply src/containers/map/index.tsx:370]: this component should not be on mobile

File: src/containers/navigation/index.tsx (hunk 2 above)
Comment (anamontiaga) [reply src/containers/navigation/index.tsx:10]: same

**Pull request closed without merging**